wagtailvideos/jinja2tags.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import, unicode_literals
+
+import mimetypes
+
+from jinja2.ext import Extension
+
+from django import template
+from django.forms.widgets import flatatt
+from django.utils.text import mark_safe
+
+
+def video(video, attrs):
+    if not video:
```

Comment:
Returning and empty string is fairly traditional, but I agree, it makes it difficult to debug. It will mean doing `{% if foo.video %}{% video foo.video %}{% endif %}` every time you want to use a video though.